.travis.yml

```diff
@@ -2,24 +2,27 @@ language:
   - node_js
 
 node_js:
-  - stable
-  - 8
-  - 6
-  - 4
+  - '10'
+  - '8'
+  - '6'
```

Comment:
Any reason to retain support for versions below 10? Or would that break linted projects that won't be migrated to node 10?